tests/unit/account_create_transaction_test.py

```
@@ -383,15 +383,15 @@ def test_create_account_transaction_with_set_alias_from_string(mock_account_ids,
 def test_create_account_transaction_with_set_alias_from_invalid_string(invalid_str):
     """Test invalid alias strings raise ValueError."""
     public_key = PrivateKey.generate().public_key()
-    tx = AccountCreateTransaction().set_key(public_key)
+    tx = AccountCreateTransaction().set_key_without_alias(public_key)
 
     with pytest.raises(ValueError):
```

Ruff (0.14.10)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)